test/unit/test_reader.cpp

```diff
@@ -0,0 +1,29 @@
+#include "../include/readers.hpp"
```

Comment:
I think the name of the header on its own should work here, without "../include/"